### PR TITLE
Added the Set Edge Crease node.

### PR DESCRIPTION
**Pull request by 3DSinghVFX**
I have added the **Set Edge Crease** node which allows to set the edge crease with AN,
Screenshot from 2020-02-16 15-30-20

Example,
Screenshot from 2020-02-16 15-30-01

### CODE REVIEW

File: animation_nodes/nodes/mesh/set_edge_crease.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import bpy
+from ... data_structures import VirtualDoubleList
+from ... base_types import AnimationNode, VectorizedSocket
+
+class EdgeCrease(bpy.types.Node, AnimationNode):
+    bl_idname = "an_SetEdgeCrease"
```
Comment (OmarEmaraDev) [inline animation_nodes/nodes/mesh/set_edge_crease.py]: `EdgeCrease` -> `SetEdgeCreaseNode`
`an_SetEdgeCrease` -> `an_SetEdgeCreaseNode`

Comment (3DSinghVFX) [reply animation_nodes/nodes/mesh/set_edge_crease.py]: Done.

**Comment by OmarEmaraDev** [comment]
Update the CHANGELOG.

**Comment by 3DSinghVFX** [comment]
> Update the CHANGELOG.

Done.

**Pull request merged**